valheim_plus.cfg

```diff
@@ -206,8 +207,13 @@ baseAutoPickUpRange=2
 ; Disable all types of camera shake
 disableCameraShake=false
 
-; The base unarmed damage multiplied by your skill level
-baseUnarmedDamage=10
+[Unarmed]
+
+enabled=false
```

Comment:
Unarmed is NOT a separate thing, it does not need to be enabled if Player not enabled